src/main/webapp/ts/matches.js

```diff
@@ -156,13 +156,16 @@ function queryServer(matchRequest) {
         var json;
         return __generator(this, function (_a) {
             json = JSON.stringify(matchRequest);
-            return [2 /*return*/, fetch('/new-matches-query', { method: 'POST', body: json })
+            return [2 /*return*/, fetch('/new-matches-query', { method: 'POST', body: json, redirect: "follow" })
                     .then(function (response) {
                     return response.json();
                 })
                     .then(function (users) {
                     //convert range from json to User
                     var out = [];
+                    if (users === '/error.html') {
```

Comment:
Is this still needed?

Reply:
If I remove it, it won't redirect.

Reply:
I have the check bc when the error hits, it returns that string so that's when I check for the redirect.

Reply:
The redirect: "follow" did not properly handle the redirect that happens when the requests are over the limit. Can I just keep this if condition to handle that for now?